### DISCUSSION TITLE
I cant access any github pages

### DISCUSSION THREAD

**Discussion by KARU0720** [community/community#132235] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, Im new here, i would like to seek some advice, i started a website last week for a online registration form, I published it and i can access it on my phone and pc, but yesterday i noticed that i cant access any websites from github. I think it is a problem from may network, because i tried using my mobile data and i can access the page.

**Comment by nguyenpham111** [comment]
try to use vpn maybe it banned from your location?
